.travis.yml

```diff
@@ -12,12 +12,15 @@ env:
   - DJANGO=1.6
   - DJANGO=1.7
   - DJANGO=1.8
-  - DJANGO==1.9rc1
+  - DJANGO=1.9
+  - DJANGO=1.10
 
 matrix:
   exclude:
     - python: 3.3
       env: DJANGO==1.9rc1
+    - python: 3.3
```

Comment:
Shouldn't DJango 1.10 be tested with Python 3.5? That seems the most common combo